test/ci_support/cdash_analyze_and_report_random_failures/rft_0_ift_2/test_queries_cache/test_history_cache/testname1_buildname1.json

```diff
@@ -0,0 +1,140 @@
+{ u'bugtracker': u'https://github.com/trilinos/Trilinos/issues',
+  u'builds': [ { u'buildName': u'build1',
+                 u'buildSummaryLink': u'build/1',
+                 u'buildstarttime': u'2018-10-28T05:56:22 UTC',
+                 u'details': u'Completed\n',
+                 u'nprocs': 4,
+                 u'prettyProcTime': u'42s 880ms',
+                 u'prettyTime': u'10s 720ms',
+                 u'procTime': 42.88,
+                 u'site': u'mutrino',
+                 u'siteLink': u'viewSite.php?siteid=223',
+                 u'status': u'Passed',
+                 u'statusclass': u'error',
```

Comment:
Here is one of the (in-theory) contradictions that I was warning you about.  I don't think it's a big deal, and probably would leave it alone since you don't use those values.  Just a warning about mocking the API responses this way.